.golangci.yml

```diff
@@ -168,3 +168,16 @@ issues:
     - path: internal/test/fixture # Don't check for magic numbers on fixtures.
       linters:
         - gomnd
+    - path: tools/signtool # Don't check copied file.
+      linters:
+        - errcheck
+        - gocritic
+        - gosec
+        - gomod
+        - gosimple
+        - noctx
+        - gomnd
+        - revive
+        - ineffassign
+        - staticcheck
+        - stylecheck
```

Comment:
this is just for one file that was copied/pasted not affecting any of the other code, the only reason why I've left this is because we do not own this code

Reply:
do you have a plan in mind to keep this file in sync with upstream? if no then we now own it